yarn.lock

```diff
@@ -4259,6 +4267,13 @@
   dependencies:
     "@types/react" "*"
 
+"@types/react-test-renderer@*":
```

Comment:
I think it could be removed yep
I added at some point react-test-renderer for implementing the unit test but it does not seems usefull at all after iterating around it. So, this PR should not bring that change. I take a look 👍🏼

Reply:
Update : 
Running dedup don't change anything, it's not related to my add/remove of react-test-renderer/react. It's more related to the addition of @testing-library/react-hooks that bring in this version (and take the latest)